postgres-appliance/launch.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
+if [ "$(id -u)" -ne 0 ]; then
+    sed -e "s/^postgres:x:[^:]*:[^:]*:/postgres:x:$(id -u):$(id -g):/" /etc/passwd > /tmp/passwd
+    cat /tmp/passwd > /etc/passwd
+    rm /tmp/passwd
+    mkdir -p "$PGROOT" "$RW_DIR/cron/crontabs"
```

Comment:
Isn't [configure_spilo.py](https://github.com/zalando/spilo/blob/master/postgres-appliance/scripts/configure_spilo.py#L790-L794) supposed to do it?